google/cloud/grpc_options.cc

```diff
@@ -43,6 +43,16 @@ grpc::ChannelArguments MakeChannelArguments(Options const& opts) {
   if (!user_agent_prefix.empty()) {
     channel_arguments.SetUserAgentPrefix(absl::StrJoin(user_agent_prefix, " "));
   }
+
+  auto constexpr kDisableKeepaliveTime =
+    std::chrono::milliseconds(std::chrono::hours(24));
+  channel_arguments.SetInt(GRPC_ARG_KEEPALIVE_TIME_MS,
```

Comment:
I think the users should be able to override these values. (maybe they are super smart and know better than to use our defaults). So I think we should only set the defaults, if there is not a value set already. Something like this:

https://github.com/googleapis/google-cloud-cpp/blob/f779bb99395bbdfe3aac51864ff2abf2a0b6dffc/google/cloud/bigtable/internal/defaults.cc#L145-L150

 (the code in `storage` did not do this, but I think it should have).